dev_guide/projects.adoc

```diff
@@ -47,6 +47,14 @@ $ oc new-project hello-openshift \
 ----
 ====
 
+ifdef::openshift-online[]
+[NOTE]
+====
+The number of projects you are allowed to create may be limited by the system administrator. Once your 
+limit is reached, you may need to delete an existing project in order to create a new one.
+====
+endif::[]
+
```

Comment:
Should this entire note be ifdef'd for Online? It would make sense instead to add an ifdef'd hyperlink from "may be limited by the system administrator" to the documentation for doing so, similar to the way we have an ifdef'd hyperlink for "If allowed" above.